addons/website_blog/views/website_blog_templates.xml

```diff
@@ -406,146 +412,163 @@
+                        <t t-set='categ_title' t-value="nav_tag_category.name"/>
+                        <t t-set='tags' t-value='nav_tag_category.tag_ids' />
+                    </t>
+                </t>
                 <t t-call="website_blog.tag_category">
-                    <t t-set='categ_title' t-value="nav_tag_category.name"/>
-                    <t t-set='tags' t-value='nav_tag_category.tag_ids' />
+                    <t t-set='hide_title' t-value='not len(tag_category)' />
+                    <t t-set='categ_title'>Others</t>
+                    <t t-set='tags' t-value='other_tags' />
                 </t>
-            </t>
-            <t t-call="website_blog.tag_category">
-                <t t-set='hide_title' t-value='not len(tag_category)' />
-                <t t-set='categ_title'>Others</t>
-                <t t-set='tags' t-value='other_tags' />
-            </t>
-        </section>
+            </div>
+        </div>
     </xpath>
 </template>
 
 <!-- Option:Right Column: archives -->
 <template id="opt_blog_rc_history" name="Archives"
         inherit_id="website_blog.index_right" active="True" customize_show="True">
     <xpath expr="//div[@id='main_column']" position="attributes">
-        <attribute name="class">col-md-8</attribute>
+        <attribute name="class">col</attribute>
     </xpath>
     <xpath expr="//div[@id='blog_right_column']" position="inside">
-        <section class="mt32">
-        <h4>Archives</h4>
-        <ul class="nav nav-pills flex-column">
-            <t t-foreach="nav_list" t-as="year">
-                <li data-toggle="collapse" t-att-data-target="'#' + year" aria-expanded="false" class="nav-item blog_post_year_collapse mt8">
-                    <i t-attf-class="fa #{'fa-chevron-down' if not date and year_index == 0 else 'fa-chevron-right'} mr4" role="img" aria-label="Next year" title="Next year"/><span t-esc="year"/>
-                </li>
-                <ul t-attf-class="blog_post_year collapse nav nav-pills flex-column #{not date and year_index == 0 and 'in'}" t-att-id="year" role="menu">
-                    <t t-foreach="nav_list[year]" t-as="months">
-                        <li t-if="months['date_begin'] == date" class="nav-item">
-                            <a role="menuitem" t-ignore="True" t-attf-href="#{blog_url(date_begin=False, date_end=False)}" class="nav-link active"><t t-esc="months['month']"/><span class="float-right badge badge-pill" t-esc="months['post_date_count']"/></a>
-                        </li>
-                        <li t-else="1" class="nav-item">
-                            <a role="menuitem" t-ignore="True" t-attf-href="#{blog_url(date_begin=months['date_begin'], date_end=months['date_end'])}" class="nav-link"><t t-esc="months['month']"/><span class="float-right badge badge-pill" t-esc="months['post_date_count']"/></a>
+        <div class="card mb-3 bg-secondary">
+            <h6 class="card-header"><b>Archives</b></h6>
```

Comment:
@kea14 You cannot use `<b>` to make titles bold, this the responsibility of the theme to say if titles are bold, so you should not force them with HTML elements.